perception/tracking_object_merger/src/decorative_tracker_merger.cpp

```diff
@@ -250,9 +250,9 @@ bool DecorativeTrackerMergerNode::decorativeMerger(
 {
   // get current time
   const auto current_time = rclcpp::Time(input_objects_msg->header.stamp);
-  if (input_objects_msg->objects.empty()) {
-    return false;
-  }
+  //  if (input_objects_msg->objects.empty()) {
+  //    return false;
+  //  }
```

Comment:
May I ask why this code is not required?
cc @YoshiRi

Reply:
With this statement, tracking object merger can not update tracklet with empty input. See https://github.com/autowarefoundation/autoware.universe/issues/6373.
It already fixed, so this part is no longer needed.